guides/micronaut-oracle-cloud/java/src/main/java/example/micronaut/HelloController.java

```diff
@@ -0,0 +1,14 @@
+package example.micronaut;
+
+import io.micronaut.http.MediaType;
+import io.micronaut.http.annotation.Controller;
+import io.micronaut.http.annotation.Get;
+
+@Controller
+public class HelloController {
+
+    @Get(value = "/", produces = MediaType.TEXT_PLAIN)
```

Comment:
@burtbeckwith I tend to like more:

```
@Produces(MediaType.TEXT_PLAIN)
@Get
```

Reply:
Why it is more verbose?

Reply:
`@Get(produces = MediaType.TEXT_PLAIN)` makes the most sense